javascript/src/methods.js

```diff
@@ -442,9 +439,15 @@ methods.addPolylines = function(polygons, layerId, group, options, popup, popupO
       .cbind(options);
 
     addLayers(this, "shape", df, function(df, i) {
-      let shape = df.get(i, "shapes")[0];
-      shape = HTMLWidgets.dataframeToD3(shape);
-      return L.polyline(shape, df.get(i));
+      let shapes = df.get(i, "shapes");
```

Comment:
It feels weird to mutate the shape array you get back from df. Instead of doing this, can you use `shapes.map(HTMLWidgets.dataframeToD3)` or similar?